rules/es6.js

```diff
@@ -28,5 +24,8 @@ module.exports = {
         'rest-spread-spacing'    : [2, 'never'],
         // Disallow spacing around embedded expressions of template strings
         'template-curly-spacing' : [2, 'never'],
+        // --- Disabled Rules ---
+        'arrow-parens'           : [0],
```

Comment:
Like you you disable this one because you don't like it :trollface:

Reply:
Just be glad I didn't turn it on and configure it the way I like it!  😈